common/src/utils/hash.ts

```
@@ -150,3 +152,35 @@ export function packBytesAndPoseidon(unpacked: number[]) {
   const packed = packBytesArray(unpacked);
   return customHasher(packed.map(String)).toString();
 }
+
+export function calculateUserIdentifierHash(
+  destChainID: number,
+  userID: string,
+  userDefinedData: string
+): BigInt {
```

Biome (1.9.4)

[error] Don't use 'BigInt' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'bigint' instead


(lint/complexity/noBannedTypes)